Topic5Mysql/MySQLFiles/high-school-DB.sql

```diff
@@ -0,0 +1,169 @@
+  `registration_number` int(11) NOT NULL AUTO_INCREMENT,
+  `date_of_birth` date NOT NULL,
+  PRIMARY KEY (`registration_number`),
+  KEY `name` (`first_name`,`last_name`)
+) ENGINE=InnoDB AUTO_INCREMENT=31 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `student`
+--
+
+LOCK TABLES `student` WRITE;
+/*!40000 ALTER TABLE `student` DISABLE KEYS */;
+INSERT INTO `student` VALUES ('Matias','Haure',1,'1991-03-05'),('Luciano','Aguirre',2,'1993-05-06'),('Pedro','Lopez',3,'1995-09-07'),('Manuel','Gimenez',4,'1999-06-08'),('Manuel','Rodriguez',5,'1999-06-08'),('Francisco','Cuenca',6,'1991-01-06'),('Emiliano','Testa',7,'1991-05-06'),('Pedro','Lupe',8,'1991-08-06'),('Ismael','Serrano',9,'1991-05-09'),('Federico','Fernandez',10,'1994-06-15'),('Lionel','Messi',11,'1988-08-14'),('Mariano','Rodecker',12,'1990-06-07'),('Matias','Varela',13,'1991-05-17'),('Luca','Otarola',14,'1992-12-14'),('Mariano','Maisonave',15,'1993-11-17'),('Franco','Toniolo',16,'1994-07-10'),('Tomas','Rossi',17,'1995-07-11'),('Juan','Stagno',18,'1991-05-01'),('Luciano','Lujan',19,'1995-11-03'),('Rogelio','Funes Mori',20,'1992-06-07'),('Amilcar','Infante',21,'1984-02-04'),('Diego','Martinez',22,'1999-05-01'),('Pepito','Locon',23,'1995-05-06'),('Walter','White',24,'1985-06-04'),('Diego','Maradona',25,'1982-05-04'),('Ismael','Serrano',26,'1994-05-09'),('Matias','Gimenez',27,'1995-04-07'),('Albano','Grosso',28,'1998-04-15'),('Lucas','Gonzales',29,'1991-10-11'),('Fernando','Cavenaghi',30,'1985-10-14');
+/*!40000 ALTER TABLE `student` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `teacher`
+--
+
+DROP TABLE IF EXISTS `teacher`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `teacher` (
+  `first_name` varchar(45) NOT NULL,
+  `last_name` varchar(45) NOT NULL,
+  `date_of_birth` date NOT NULL,
+  `teacher_id` int(11) NOT NULL AUTO_INCREMENT,
+  PRIMARY KEY (`teacher_id`),
+  KEY `name` (`first_name`,`last_name`)
+) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `teacher`
+--
+
+LOCK TABLES `teacher` WRITE;
+/*!40000 ALTER TABLE `teacher` DISABLE KEYS */;
+INSERT INTO `teacher` VALUES ('Alejandro','Stankevicius','1985-11-02',1),('Diego','Martinez','1984-02-01',2),('Karina','Cenci','1983-06-01',3);
+/*!40000 ALTER TABLE `teacher` ENABLE KEYS */;
+UNLOCK TABLES;
+/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;
+
+/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
+/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
+/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
+/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
+/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
+/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
+
+-- Dump completed on 2015-03-05 19:11:19

```

Comment:
sure no problem. The changes I made were basically indexing each table with significant attributes for it.